Dockerfile

```diff
@@ -1,10 +1,22 @@
-FROM alpine:3.5
+FROM golang:1.12-alpine as builder
 
-MAINTAINER wata727
+RUN apk --no-cache add git make gcc musl-dev zip
+
+WORKDIR /go/src/github.com/wata727/tflint/
+
+ADD . /go/src/github.com/wata727/tflint
+
+RUN make release
+
+RUN find /go/src/github.com/wata727/tflint/dist/
+
+FROM alpine:3.9 as prod
+
+LABEL maintainer=wata727
 
 RUN apk add --no-cache ca-certificates
 
-COPY dist/linux_amd64/tflint /usr/local/bin
+COPY --from=0 /go/src/github.com/wata727/tflint/dist/linux_amd64/tflint /usr/local/bin
```

Comment:
I guess that `--from=builder` is more easy to understand.

Reply:
updated